cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/k8s/ServiceOptions.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.app.runtime.k8s;
+
+import co.cask.cdap.common.options.Option;
+import co.cask.cdap.master.environment.k8s.EnvironmentOptions;
+
+/**
+ * Environment options for application service programs.
+ */
+public class ServiceOptions extends EnvironmentOptions {
+  public static final String APP_SPEC_PATH = "appSpecPath";
+  public static final String PROGRAM_OPTIONS_PATH = "programOptions";
+  public static final String TMP_DIR = "tmpDir";
+  public static final String BIND_ADDRESS = "bindAddress";
+
+  @Option(name = APP_SPEC_PATH, usage = "Path to application specification file")
+  private String appSpecPath;
+
+  @Option(name = PROGRAM_OPTIONS_PATH, usage = "Path to program options file")
+  private String programOptionsPath;
+
+  @Option(name = TMP_DIR, usage = "Directory for temporary files")
+  private String tmpDir;
```

Comment:
Why this is needed? Is it for testing? Also, since you can already customize the conf via the `--conf` option, it's better to use `CFG_LOCAL_DATA_DIR`.

Reply:
yeah it was just for testing. Added this before the --conf option will there, will just use the config from the cconf.